### PR TITLE
Ubuntu 20.04 (clean) requirements

### PR DESCRIPTION
**Pull request by AbelLykens**
After https://github.com/Coldcard/firmware/pull/173 I started a new clean Ubuntu 20.04. This PR adds on what I missed, and explains steps a bit more + includes command for running outside of virt environment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,36 +135,47 @@ See <https://bugs.python.org/issue32909>
...
 
 ```shell
-apt install build-essential git python3 python3-pip libudev-dev gcc-arm-none-eabi libffi-dev xterm swig libpcsclite-dev python-is-python3 autoconf libtool
-```
+# Install (system) requirements, tools and libraries
+apt install build-essential git python3 python3-pip libudev-dev gcc-arm-none-eabi libffi-dev xterm swig libpcsclite-dev python-is-python3 autoconf libtool python3-venv
 
-Install and run simulator on Ubuntu 20.04
+# Install all in home directory (optional but useful later)
+cd ~
```
Comment (doc-hex) [inline README.md:146]: IDK. that's quite personal... maybe leave out these two lines

File: README.md [hunk 2]
```diff
@@ -135,36 +135,47 @@ See <https://bugs.python.org/issue32909>
...
 make ngu-setup
 make
-# below line runs the simulator
+
+# Run the simulator in the active virtualenv
 ./simulator.py
+
+# Later, if you want to run it (after a reboot):
+cd ~/firmware/unix/
+~/firmware/ENV/bin/python3 ./simulator.py
```
Comment (doc-hex) [inline README.md:178]: better to activate the virtenv, then run it.

**Comment by AbelLykens** [comment]
Ack -- will edit, sec. Also: is there a way to start the simulator as if the CC was just unwrapped? Showing setup instructions?

**Comment by AbelLykens** [comment]
_Edits made_

**Comment by doc-hex** [comment]
> Ack -- will edit, sec. Also: is there a way to start the simulator as if the CC was just unwrapped? Showing setup instructions?

Yes, I think `-w` is the option you want. Many others, see [unix/readme.md](https://github.com/Coldcard/firmware/blob/master/unix/README.md)

**Pull request merged**